### PR TITLE
feat: add zh-tw translation for DAO job board project.

### PR DESCRIPTION
**Pull request by moojing**
## Overview
- Add Chinese translation to job board. 
- add `packages/client/public/locales/zh/common.json`.

## Related PR 
- https://github.com/Developer-DAO/DAO-job-board/issues/179

### CODE REVIEW

File: packages/client/public/locales/zh/common.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          "category_error": "請確認至少有選擇三個關鍵字",
+          "keyword_button_text": "選擇關鍵字",
+          "compensation_header": "薪酬說明 (選填)",
+          "compensation_sub_header": "有薪資資訊的工作職位容易獲得更多的申請",
+          "token_default": "代幣 / 貨幣",
+          "equity_header": "您提供股權嗎? (選填)",
+          "equity_default": "權益範圍",
+          "location_header": "地點 (選填)",
+          "location_sub_header": "預設為遠端工作",
+          "location_placeholder": "如： 美國加勒佛尼亞州",
```
Comment (Yukaii) [inline packages/client/public/locales/zh/common.json:69]: `加勒佛尼亞` or `加利佛尼亞`？I'm not sure which is more common for Chinese users.

File: packages/client/public/locales/zh/common.json [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    },
+    "forms": {
+      "post_job": {
+        "continue_button_text": "繼續",
+        "cancel_button_text": "取消",
+        "back_button_text": "上一步",
+        "job_basics": {
+          "header": "工作職位基本資訊",
+          "sub_header": "讓我們知道你正在尋找哪方面的專業人才",
+          "title_header": "請為你的職位輸入相符的標題",
```
Comment (Yukaii) [inline packages/client/public/locales/zh/common.json:42]: Sounds a little bit weird

Comment (moojing) [reply packages/client/public/locales/zh/common.json]: ok, I've change `相符` to `清楚`，it's more matched to the word `clear`.
https://github.com/Developer-DAO/DAO-job-board/pull/185/commits/84043b2fbcaeb6d392c1d8dea984a57940150f02

File: packages/client/public/locales/zh/common.json (hunk 1 above)
Comment (moojing) [reply packages/client/public/locales/zh/common.json]: I've change the value to `加利佛尼亞`
https://github.com/Developer-DAO/DAO-job-board/pull/185/commits/84043b2fbcaeb6d392c1d8dea984a57940150f02

**Review by carlomigueldy — APPROVED** [review]
Thanks for contributing my friend, ang apologies for the late review! @moojing 🎉 

@allcontributors please add @moojing for code, translation

**Comment by carlomigueldy** [comment]
@allcontributors please add @moojing for code, translation

**Comment by carlomigueldy** [comment]
@allcontributors please add @Yukaii for review, translation

**Comment by moojing** [comment]
@carlomigueldy  np, thank you for help reviewing. I'll keep finding what I can do here.

**Pull request merged**